app/controllers/mentees_controller.rb

```diff
@@ -0,0 +1,11 @@
+class MenteesController < ApplicationController
+  def index
+    mentees = User.where(user_type: "mentee")
+
+    if mentees.empty?
+      render json: { message: "No mentee recorded" }, status: 404
+    else
+      render json: { mentees: mentees }, status: 200
+    end
```

Comment:
This is fine, though another way be
```ruby
status: :not_found
```
and
```ruby
status: :ok
```
You may not even explicitly specify ok since it is the default, though it should still be tested!

Reply:
Ok. Thanks for the suggestions. I have implemented it.